code/game/machinery/_machinery.dm

```diff
@@ -119,10 +119,9 @@ Class Procs:
 	var/critical_machine = FALSE //If this machine is critical to station operation and should have the area be excempted from power failures.
 	var/list/occupant_typecache //if set, turned into typecache in Initialize, other wise, defaults to mob/living typecache
 	var/atom/movable/occupant = null
-	var/unsecuring_tool = /obj/item/wrench
 	var/interact_open = FALSE // Can the machine be interacted with when in maint/when the panel is open.
 	var/interact_offline = 0 // Can the machine be interacted with while de-powered.
-	var/speed_process = 0 // Process as fast as possible?
+	var/speed_process = FALSE // Process as fast as possible?
```

Comment:
I believe I set this the right way around. Shout at me if I am wrong. Or just shout at me.

Reply:
yes FALSE = 0 is a define in our code and is the clear and preferred binary variable